tests/std/tests/P1147R1_printing_volotile_pointers/test.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cstddef>
+#include <sstream>
+
+using namespace std;
+
+std::string getExpected(int* ptr) {
+    ostringstream out;
+    out << ptr;
+    return out.str();
+}
+
+std::string getActual(volatile int* ptr) {

```

Comment:
These two functions are identical except for parameter type - they could easily be merged into a single function template.